setup.cfg

```diff
@@ -10,7 +10,7 @@ all_files = 1
 upload-dir = docs/build/html
 
 [bdist_rpm]
-requires = Django >= 1.11
+requires = Django >= 2.0
 
 [wheel]
 universal = 1
```

Comment:
The wheel is no longer universal, you should drop the `[wheel]` section entirely.